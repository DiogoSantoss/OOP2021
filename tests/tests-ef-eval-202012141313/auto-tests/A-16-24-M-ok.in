8
2
S2
1
B
10
4
0
0
6
2
0
8
2
S2
6
B
1
2
S2
4
BK
1
2
S2
10
C
1
1
1
1
2
1
3
0
6
2
0
4
4
8
1
1
1
2
1
3
0
0
