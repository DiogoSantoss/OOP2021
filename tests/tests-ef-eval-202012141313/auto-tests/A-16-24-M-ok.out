Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0
S2|Ricardo Figueiredo|Lisboa|ELITE|4000|4000
Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: 1|S2|B|1|400|360|6
Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: 2|S2|BK|1|200|180|4
Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: 3|S2|C|1|300|270|10
Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0
S2|Ricardo Figueiredo|Lisboa|ELITE|4900|4000
Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Número de dias a avançar: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: 1|S2|B|1|400|360|6
Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: 2|S2|BK|1|200|180|4
Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: 3|S2|C|1|300|270|10
Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: 